LibraBFT/Yasm/System.agda

```diff
@@ -252,6 +252,12 @@ module LibraBFT.Yasm.System
                         → peerStates (StepPeer-post theStep) ≡ peerStates pre
  cheatStepDNMPeerStates {pid = pid} {pre = pre} (step-cheat _ _) _ = override≡Correct {f = peerStates pre} {pid}
 
+ cheatStepDNMPeerStates₁ : ∀{e pid pid' st' outs}{pre : SystemState e}
+                        → (theStep : StepPeer pre pid st' outs)
+                        → isCheat theStep
+                        → peerStates (StepPeer-post theStep) pid' ≡ peerStates pre pid'
+ cheatStepDNMPeerStates₁ {_} {pid} {pid'} (step-cheat fm x₁) x = overrideAux pid pid'
```

Comment:
If we're going to use `overrideAux` directly, let's give it a better name.  Maybe `overrideSameVal-correct`, and rename `override≡Correct` to `overrideSameVal-correct-ext` (the extensional version of the same property)?